### PR TITLE
fix(query): fix upload variant data type

### PR DESCRIPTION
**Pull request by b41sh**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

fix upload variant data type

- Closes #issue

### CODE REVIEW

File: tests/suites/1_stateful/05_formats/05_05_parquet/05_05_01_parquet_load_unload.sh [hunk 1]
```diff
@@ -29,9 +29,6 @@ test_format() {
 
 	echo "truncate table test_load_unload" | $MYSQL_CLIENT_CONNECT
 
-	curl -s -u root: -XPOST "http://localhost:${QUERY_CLICKHOUSE_HTTP_HANDLER_PORT}" \
```
Comment (youngsofun) [inline tests/suites/1_stateful/05_formats/05_05_parquet/05_05_01_parquet_load_unload.sh:32]: why remove this row?

I forgot that load_unload did not cover copy,  it only tested clickhouse handler and streaming load.
they used diff code path, we should add test for copy.

**Comment by b41sh** [comment]
askbend:summary

File: tests/suites/1_stateful/05_formats/05_05_parquet/05_05_01_parquet_load_unload.sh [hunk 2]
```diff
@@ -25,24 +25,31 @@ test_format() {
...
+	echo "copy into test_load_unload from 'fs:///tmp/test_load_unload.parquet' file_format = (type = parquet);" | $MYSQL_CLIENT_CONNECT
+
+	# unload clickhouse again
+	curl -s -u root: -XPOST "http://localhost:${QUERY_CLICKHOUSE_HTTP_HANDLER_PORT}" \
+	-d "select * from test_load_unload FORMAT ${1}" > /tmp/test_load_unload3.parquet
 
-	diff /tmp/test_load_unload2.txt /tmp/test_load_unload.txt
-	rm  /tmp/test_load_unload2.txt /tmp/test_load_unload.txt
+	diff /tmp/test_load_unload2.parquet /tmp/test_load_unload.parquet
+	diff /tmp/test_load_unload3.parquet /tmp/test_load_unload.parquet
```
Comment (youngsofun) [inline tests/suites/1_stateful/05_formats/05_05_parquet/05_05_01_parquet_load_unload.sh:51]: ok we have ` copy into table`, better check `copy into  stage` too

Comment (youngsofun) [reply tests/suites/1_stateful/05_formats/05_05_parquet/05_05_01_parquet_load_unload.sh:51]: rest lgtm

**Review by youngsofun — APPROVED** [review]

**Pull request merged**